docs/Cliente REST/2-Axios.md

```diff
@@ -25,29 +24,27 @@ Primeiramente, devemos compreender que o termo API é muito amplo e tem inúmera
 
 Antes de colocamos a mão no código devemos saber que uma API HTTP tem regras. Dependendo da situação será necessário usar um método **X** e em outro momento **Y**. Então, que tal conhecer esses métodos?
 
-`GET`: Com o método GET solicitamos dados de um recurso específico
-`POST`: Com o método POST podemos realizar interações na API
-`DELETE`: Com esse método podemos apagar dados via API
-`PUT`: Com esse método podemos alterar dados da API. Ele atualiza recursos já existentes ou pode criar um recurso inexistente
+`GET`: Com o método GET solicitamos dados de um recurso específico  
+`POST`: Com o método POST podemos realizar interações na API  
+`DELETE`: Com esse método podemos apagar dados via API  
+`PUT`: Com esse método podemos alterar dados da API. Ele atualiza recursos já existentes ou pode criar um recurso inexistente  
 `PATCH`: Esse método é parecido com o `PUT`. No entanto, ele altera dados específicos
 
-Exemplos:
-
-`GET`: Listar todos os usuários registrados
-`POST`: Registrar um novo usuário
-`DELETE`: Apagar um ou mais usuários
-`PUT`: Alterar os dados de um usuário, caso não encontre o usuario ele é criado.
+Exemplos:  
+`GET`: Listar todos os usuários registrados  
+`POST`: Registrar um novo usuário  
+`DELETE`: Apagar um ou mais usuários  
+`PUT`: Alterar os dados de um usuário, caso não encontre o usuario ele é criado.  
 `PATCH`: Alterar um dado do usuário, como por exemplo a senha
 
 Tambem existem outros métodos que não são usados tanto em uma API. Caso queira dar uma conferida numa lista completa, [clique aqui](https://developer.mozilla.org/pt-BR/docs/Web/HTTP/Methods).
 
-
 Alem dos métodos apresentados, temos os status da API. Eles servem para informar para o desenvolvedor o que aconteceu com sua requisição.
 
-Por exemplo:
-`201`: Seu pedido foi aceito e foi criado na API
-`200`: Tudo certo com o seu pedido
-`400`: Não deu para entender seu pedido, pode ser que há uma falha na estrutura nele
+Por exemplo:  
+`201`: Seu pedido foi aceito e foi criado na API  
+`200`: Tudo certo com o seu pedido  
+`400`: Não deu para entender seu pedido, pode ser que há uma falha na estrutura nele  
```

Comment:
Mesma coisa. Esses espaços tem alguma motivação ou vieram sem querer? 

```suggestion
Por exemplo:
`201`: Seu pedido foi aceito e foi criado na API
`200`: Tudo certo com o seu pedido
`400`: Não deu para entender seu pedido, pode ser que há uma falha na estrutura nele
```

Reply:
**Atualmente está assim no GitHub:**
![css-in-js01](https://user-images.githubusercontent.com/23555768/97715352-0ae88700-1aa1-11eb-8dfe-c537da1e630f.png)

**O código do arquivo está assim:**
![css-in-js02](https://user-images.githubusercontent.com/23555768/97715354-0b811d80-1aa1-11eb-84a9-0ad0eca5a70b.png)

**Coloquei as duas linhas no final do parágrafo para quebrar a linha e ficar assim:**
![css-in-js03](https://user-images.githubusercontent.com/23555768/97715355-0b811d80-1aa1-11eb-9e01-5c875e3e4f15.png)